tools/pot/openvino/tools/pot/algorithms/quantization/bias_correction/algorithm.py

```diff
@@ -270,15 +274,15 @@ def _create_parameters_for_input_nodes(input_nodes):
             inputs_data.append({
                 'param_name': param_node.name,
                 'param_shape': tuple(c_input_shape),
-                'input_name': input_node.name
+                'input_name': input_node_name
             })
 
         return inputs_data
 
     def _create_results_after_nodes(self, output_nodes):
         outputs_data = []
         for output_node in output_nodes:
-            output_name = output_node.name
+            output_name = nu.create_node_name(output_node, str)
```

Comment:
What do you think about always return tuple name in  `nu.create_node_name` and after that call https://github.com/openvinotoolkit/openvino/blob/2e290a6e72107791c9412cc8a94cb6c45323daa6/tools/pot/openvino/tools/pot/utils/utils.py#L23 to create IE-like name? 

cc @AlexKoff88, @KodiaqQ

Reply:
Good idea to have an IE-like name for statistics everywhere because this way is used in the new IE API for output adding (see reference [at this draft PR](https://github.com/openvinotoolkit/openvino/pull/9004/files#diff-593772cbb618bbd1dc6c2a9c09060b9f1261d0baeeaf4bb94e168b38a95277c3R280)).
But it looks like we will need to update POT globally to support this format.

Reply:
I mean, can we rewrite it like this
```python
output_name = nu.create_node_name(output_node)
output_name = convert_output_key(output_name)
```